src/main/java/frc/team4909/robot/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -0,0 +1,37 @@
+package frc.team4909.robot.subsystems.elevator;
+
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.PWMTalonSRX;
+import edu.wpi.first.wpilibj.PWMVictorSPX;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class ElevatorSubsystem extends Subsystem{
+    PWMVictorSPX leftElevatorSPX, rightElevatorSPX;
+    PWMTalonSRX leftElevatorSRX, rightElevatorSRX, pivotSRX;
+    Encoder pivotEncoder, liftEncoder1, liftEncoder2;
+    DigitalInput topLimitSwitch, bottomLimitSwitch;
+
+    public void ElevatorSubsytem(){
+        //Intake Pivot
+        pivotSRX = new PWMTalonSRX(3);
+        pivotEncoder = new Encoder(1, 2);
+
+        //Lift
+        leftElevatorSPX = new PWMVictorSPX(4);
+        rightElevatorSPX = new PWMVictorSPX(5);
+        leftElevatorSRX = new PWMTalonSRX(6);
+        rightElevatorSRX = new PWMTalonSRX(7);
+        liftEncoder1 = new Encoder(3, 4);
+        liftEncoder2 = new Encoder(5, 6);
+        topLimitSwitch = new DigitalInput(7);
```

Comment:
no limit switches anymore, it's continuous top to bottom – we should check with build on how we plan to "zero" out, whether that be starting at a given height, with a clear distinction on the robot itself, or somehow managing to use a magnetic limit switch